configs/default/TURC-FPVCYCLEF401.config

```diff
@@ -0,0 +1,73 @@
+# Betaflight / STM32F411 (S411) 4.2.3 Sep 20 2020 / 20:43:12 (2696b7c88) MSP API: 1.43
+
+board_name FPVCYCLEF401
+manufacturer_id TURC
+
+# resources
+resource BEEPER 1 B02
+resource MOTOR 1 B04
+resource MOTOR 2 B05
+resource MOTOR 3 B06
+resource MOTOR 4 B07
+resource PPM 1 A03
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource LED 1 C13
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource ADC_BATT 1 B00
+resource OSD_CS 1 B12
+resource GYRO_EXTI 1 A01
+resource GYRO_CS 1 A04
+resource USB_DETECT 1 C15
+
+# timer
+timer A03 AF3
+# pin A03: TIM9 CH2 (AF3)
+timer B04 AF2
+# pin B04: TIM3 CH1 (AF2)
+timer B05 AF2
+# pin B05: TIM3 CH2 (AF2)
+timer B06 AF2
+# pin B06: TIM4 CH1 (AF2)
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer A02 AF2
+# pin A02: TIM5 CH3 (AF2)
+
+# dma
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
+dma pin B04 0
+# pin B04: DMA1 Stream 4 Channel 5
+dma pin B05 0
+# pin B05: DMA1 Stream 5 Channel 5
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin A02 0
+# pin A02: DMA1 Stream 0 Channel 6
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature OSD
+
+# master
+set motor_pwm_protocol = DSHOT600

```

Comment:
Will this target only be used for an all-in-one offering that comes with on-board ESCs that can handle Dshot600?
If not then using this as a default will lead to the risk of unintended motor spin up if the flight controller is used with certain ESCs.

Furthermore, if this is left at the default, the design of Betaflight configurator is such that the user is prompted to select an appropriate ESC protocol on connect, making this a good user experience, and not likely to result in support requests:

![image](https://user-images.githubusercontent.com/4742747/101479127-8e599a00-39b6-11eb-9a6a-b46a159d9181.png)

Reply:
Yes, this FC comes with on-board ESCs that can handle Dshot600.